x-pack/plugins/enterprise_search/public/applications/app_search/components/engine/engine_router.tsx

```diff
@@ -69,31 +69,30 @@ export const EngineRouter: React.FC = () => {
     },
   } = useValues(AppLogic);
 
+  const { engineName: engineNameFromUrl } = useParams() as { engineName: string };
```

Comment:
Feel free to push back if you disagree, but I was thinking `FromUrl` flowed a little better reading-wise and also made it a little clearer where it's coming from and where we're setting engineName from. I also played with `engineNameFromParam` but ended up likeing `FromUrl` a little better